karton/mwdb_reporter/mwdb_reporter.py

```diff
@@ -71,268 +92,408 @@ def mwdb(self) -> MWDB:
             mwdb.login(mwdb_config["username"], mwdb_config["password"])
         return mwdb
 
-    def _tag_children_blobs(self, config: MWDBConfig) -> None:
-        for item in config.config.values():
-            if type(item) is MWDBBlob:
-                tag = config.family
+    @property
+    def mwdb(self) -> MWDB:
+        if not hasattr(self, "_mwdb"):
+            setattr(self, "_mwdb", self._get_mwdb())
+        return getattr(self, "_mwdb")
+
+    def _add_tags(self, mwdb_object: MWDBObject, tags: List[str]):
+        # Upload tags and attributes via subsequent requests
+        for tag in tags:
+            if tag not in mwdb_object.tags:
                 self.log.info(
-                    "[blob %s] adding family tag '%s' inherited from parent sample",
-                    item.id,
+                    "[%s %s] Adding tag %s", mwdb_object.TYPE, mwdb_object.id, tag
+                )
+                mwdb_object.add_tag(tag)
+            else:
+                self.log.info(
```

Comment:
We should delete this log or add a similar one for attributes and comments

Reply:
There is similar one for attributes. Comments are always added incrementally.